scripts/getStatus.js

```diff
@@ -111,12 +112,23 @@ async function runScript() {
 		updateResult(swagger, validationErrors, true);
 	  }
   }
-  
+  console.log(`\t- Running Linter.`);
+
   for (let readme of readmesToProcess) {
-	  console.log(`Configuration file: "${readme}"`);
-      const linterErrors = await runLinter(readme);
+	  console.log(`Linter Validation on configuration file: "${readme}"`);
+      let linterCmd = 'autorest ' + readme + ' --azure-validator=true --message-format=json';
```

Comment:
Can you please add --Validation to this command? So it will be
`'autorest ' + readme + ' --azure-validator=true --Validation --message-format=json';`

This takes in Validation section into consideration for readme's that include it.

Reply:
Is this just for the linter command?  The model validation command is the new one below.  I'll update per your suggestion but let me know if the model command needs updating as well.  Thanks!

Reply:
@kirthik @olydis can you remind me what the validation switch is for? I don't see it at https://github.com/Azure/autorest/blob/master/docs/user/cli.md